workflows/offline_ml_diags/offline_ml_diags/create_report.py

```diff
@@ -27,6 +29,7 @@
     drop_temperature_humidity_tendencies_if_not_predicted,
 )
 from ._select import plot_transect
+from .compute_diags import METADATA_JSON_NAME
```

Comment:
Since these are in the same package I'll change them all to imports. If we don't use these files other than to make this report, these should probably be unified into one script without IO between them.

Reply:
Done.